ci/Jenkinsfile

```diff
@@ -1,39 +1,134 @@
-pipeline {
-    agent { label 'permaplant' }
 
-    stages {
-        stage('Build PR') {
-            environment {
-                DATABASE_URL = 'postgres://cidb:cidb@127.0.0.1/cidb'
+stage('Build Schema'){
```

Comment:
The solution to your fmt check simply is to do this later. Btw. I cannot reproduce it, for me src/schema.rs successfully passes `cargo fmt --check`.

Reply:
@markus2330  It really sounded like a great idea, unfortunately we cannot run `cargo fmt` without having `schema.rs`.

```bash
lukas@elektravm:~/Developer/permaplant/backend$ rm src/schema.rs 
lukas@elektravm:~/Developer/permaplant/backend$ cargo fmt
Error writing files: failed to resolve mod `schema`: /home/lukas/Developer/permaplant/backend/src/schema.rs does not exist
```

Reply:
Should have tried first :roll_eyes: 

There is an easy workaround, though:

```
touch src/schema.rs
cargo fmt
```